src/zeit/content/article/edit/browser/edit.py

```diff
@@ -128,16 +128,24 @@ def writeable(self):
     def sortable(self):
         return zope.security.canAccess(self.context, 'updateOrder')
 
+    def values(self):
+        return self.context.values()
+
     @property
     def body_css_class(self):
         css_class = ['editable-area']
         if self.sortable:
             css_class.append('action-block-sorter')
         return ' '.join(css_class)
 
-    @property
-    def values(self):
-        return self.context.values()
+
+class EditableHeaderArea(object):
+    def show_area(self):
```

Comment:
So ganz ohne die Area geht es nicht, weil dann wohl keine ID mehr existiert in die hineinersetzt werden kann. Hoffe so gehts auch. (Wird im PT ausgewertet)